### ISSUE TITLE
Native row is created by moving our blocks

### ISSUE THREAD

**Issue by Olekrut** [maxi-blocks/maxi-blocks#5611]
### Description

Native row is creating by moving our blocks. It is one of the cases that i can re-produce, 

### Step-by-step reproduction instructions

move right button to left button, check tree

re-produce steps from video

### Screenshots, screen recording, code snippet

https://jam.dev/c/6f028583-d6f2-4608-9ec7-87f502d876ce

### URL and/or Code editor

[bug with creating native row.txt](https://github.com/user-attachments/files/15816913/bug.with.creating.native.row.txt)

**Comment by myroslavve** [comment]
Cannot reproduce this, @Olekrut, can you still reproduce this?

**Comment by Olekrut** [comment]
@myroslavve 
yes, I can on master, use code above and re-produce steps from video
https://jam.dev/c/d4bbc4d3-cffc-40dd-b1ce-ae3f6bfb80f9

**Comment by kyrapieterse** [comment]
seems fixed
